proto/proto/store/rpc.proto

```diff
@@ -232,15 +242,23 @@ message AccountStorageDetails {
 // CHECK NULLIFIERS
 // ================================================================================================
 
-// List of nullifiers to return proofs for.
+// List of nullifiers to check for inclusion or non-inclusion proofs.
```

Comment:
Nit: this comment is overly specific since this is just a list of nullifiers. The _usage_ in `CheckNulifiers` adds the context about proofs etc.